CHANGES.md

```diff
@@ -1,3 +1,18 @@
+## edge-19.4.1
+
+* CLI
+  * Introduced an `upgrade` command! This allows an existing Linkerd Control Plane to be reinstalled or reconfigured
+  * The `inject` command proxy options are now converted into config annotations; the annotations ensure that these configs are persisted in subsequent resource updates
+  * Always show open TCP connections in the CLI
+  * The `stat` command now always shows the number of open TCP connections
```

Comment:
Is this a duplicate of the previous item?

Reply:
Ah yes good catch -- thanks!